src/GitVersion.Core/Core/RepositoryStore.cs

```diff
@@ -158,7 +155,7 @@ public IEnumerable<IBranch> GetBranchesForCommit(ICommit commit)
     public IEnumerable<IBranch> GetExcludedInheritBranches(Config configuration)
         => this.repository.Branches.Where(b =>
         {
-            var branchConfig = configuration.GetConfigForBranch(b.Name.WithoutRemote);
+            var branchConfig = configuration.FindConfigForBranch(b.Name.WithoutRemote);
```

Comment:
I saw there are some renamed methods called _sometingConfig_ some _somethingConfiguration_. I think we need to agree to a naming convention for these methods, and I'd suggest the Configuration name to be used

Reply:
I agree.

Reply:
I agree with you. In my opinion abbreviation should be avoided. Better to name it configuration.

But we have some conflicts here if I think about the ConfigBuilder and the ConfigurationBuilder. Both have different usage:

1. The ConfigurationBuilder is used to create the Config class for productive.
2. The ConfigBuilder is used for unit and integration tests.

Reply:
Also I use Find for a method if the result can be null. If not then I use normally Get

Reply:
In the v6 refactoring the GetExcludedInheritBranches method will be removed and the FindConfigForBranch (at this moment internal) will be marked as private. The new GetBranchConfiguration should be used instead.

Reply:
Done